AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/editor_physx_collider.py

```diff
@@ -0,0 +1,499 @@
+
+    # Shape: Cylinder
+    def set_cylinder_shape(self) -> None:
+        """
+        Property Type, Default Visibility - ('EditorProxyCylinderShapeConfig', 'NotVisible')
+
+        Used to set the PhysX Collider's Shape property to Cylinder.
+        """
+        self.component.set_component_property_value(self.Path.SHAPE, physics.ShapeType_Cylinder)
+
+    def set_cylinder_subdivision(self, subdivisions: int) -> None:
+        """
+        Property Type, Default Visibility - ('unsigned char', 'Visible')
+
+        Used to set the PhysX Collider's Cylinder Shape's subdivision. Subdivision supports int values [3-125].
+        """
+        _validate_property_visibility(self.component, self.Path.Cylinder.SUBDIVISION, PropertyVisibility.VISIBLE)
+
+        self.component.set_component_property_value(self.Path.Cylinder.SUBDIVISION, subdivisions)
+
+    def get_cylinder_subdivision(self) -> int:
+        """
+        Property Type, Default Visibility - ('unsigned char', 'Visible')
+
+        Used to get the PhysX Collider's Cylinder Shape's subdivision. Subdivision supports int values [3-125].
+        """
+        return self.component.get_component_property_value(self.Path.Cylinder.SUBDIVISION)
+
+    def set_cylinder_height(self, height: float) -> None:
+        """
+        Property Type, Default Visibility - ('float', 'Visible')
+
+        Used to set the PhysX Collider's Cylinder Shape's height.
+        """
+        assert isinstance(height, float), f"The value passed to Cylinder Height \"{height}\" is not a float."
+        _validate_property_visibility(self.component, self.Path.Cylinder.HEIGHT, PropertyVisibility.VISIBLE)
+
+        self.component.set_component_property_value(self.Path.Cylinder.HEIGHT, height)
+
+    def get_cylinder_height(self) -> float:
+        """
+        Property Type, Default Visibility - ('float', 'Visible')
+
+        Used to get the PhysX Collider's Cylinder Shape's height.
+        """
+        return self.component.get_component_property_value(self.Path.Cylinder.HEIGHT)
+
+    def set_cylinder_radius(self, radius: float) -> None:
+        """
+        Property Type, Default Visibility - ('float', 'Visible')
+
+        Used to set the PhysX Collider's Cylinder Shape's radius.
+        """
+        assert isinstance(radius, float), f"The value passed to Cylinder Radius \"{radius}\" is not a float."
+        _validate_property_visibility(self.component, self.Path.Cylinder.RADIUS, PropertyVisibility.VISIBLE)
+
+        self.component.set_component_property_value(self.Path.Cylinder.RADIUS, radius)
+
+    def get_cylinder_radius(self) -> float:
+        """
+        Property Type, Default Visibility - ('float', 'Visible')
+
+        Used to get the PhysX Collider's Cylinder Shape's radius.
+        """
+        return self.component.get_component_property_value(self.Path.Cylinder.RADIUS)
+
+    # Shape: PhysicsAsset
+    def set_physicsasset_shape(self) -> None:
```

Comment:
is there a get_physicsasset_shape? does it make sense to get the property value?

Reply:
If there were a getter for shape, it would be `get_shape`.

Since setting the shape sets the visibility of the shape related properties, I opted not to add a `get_shape`. However, I've  been contemplating creating one. Was easy enough to do so I added it.